Update Getting Started (#231)
ankitects/anki-manual: src/getting-started.md

### Cards

A question and answer pair is called a _card_. It's similar to a paper

> flashcard with a question on front and answer on the back. However, in
> Anki, a card doesn't actually look like a physical card, and when you
> look at the answer the question remains visible by default. For example, if

Reviewer: ```suggestion
flashcard with a question on the front and an answer on the back. However, in
Anki, a card doesn't look like a physical card, and when you
show the answer the question remains visible by default. For example, if
```

- Edited for grammar.
- Using both "however" and "actually" was too much. 
- Using both "look like" and "look at" was too much. "Show" is a good word to use there, since that's what the button says. "Reveal" is another possibility.

you're studying basic chemistry, you might see a question like:

    Q: Chemical symbol for oxygen?

After deciding the answer is O, you click the
"Show Answer" button, and Anki shows you:

    Q: Chemical symbol for oxygen?
    A: O

After confirming that you are correct, you tell Anki how well you

> remembered the answer, and Anki will choose when to show you the card again. If, for example, Anki decides to show you the card again in 30 days, we say the card has a 30 day interval.

Reviewer: ```suggestion
remembered the answer, and Anki will choose when to show you the card again. For example, if Anki decides to show you the card again in 3 days, we say the card has a 3 day interval.
```
30d will seem like an eternity to a new user. I picked 3 arbitrarily, but anything 2-5 would probably be just as good.

> remembered the answer, and Anki will choose when to show you the card again. If, for example, Anki decides to show you the card again in 30 days, we say the card has a 30 day interval.

Author: I'm not sure how to do this properly. The manual doesn't explicitly explain intervals anywhere but we need it here as defining young and mature later needs understanding interval.

"For example" isn't a good way to transition to this, which is why I started with a "If". I agree with 30d versus 3d but would get rid of the hyphen.

> remembered the answer, and Anki will choose when to show you the card again. If, for example, Anki decides to show you the card again in 30 days, we say the card has a 30 day interval.

Reviewer: Oh yes, I see what you mean about needing to transition it better, but I think the multiple commas makes the sentence stutter getting off the ground. 

If you want to define it here -- how about doing it without an example? 
```suggestion
remembered the answer, and Anki will choose when to show you the card again. The time between now and when Anki schedules the card for you to study it again is called the card's _interval_.
```

> remembered the answer, and Anki will choose when to show you the card again. If, for example, Anki decides to show you the card again in 30 days, we say the card has a 30 day interval.

Author: I think with example is better; the definition isn't that easy for me. This might not be that bad though:

```suggestion
remembered the answer, and Anki will choose when to show you the card again. For example, Anki might decide to show you the card again in 3 days. In this case, we say the card now has a 3 day interval.
```

> remembered the answer, and Anki will choose when to show you the card again. If, for example, Anki decides to show you the card again in 30 days, we say the card has a 30 day interval.

Reviewer: That works. [And I know you don't like the hyphen, but it's a compound adjective before a noun, so [that's the rule](https://www.grammarly.com/blog/compound-adjectives/).]

- **Review:** Cards that were previously learnt, and now need to be timely reviewed so you don’t forget them.
  There are two types of review cards:
    - **Young:** A young card is one that has an interval of less than 21 days.
    - **Mature:** A mature card is one that has an interval of 21 days or greater.

- **Relearn:** A relearning card is a card that you have failed in review mode, thus
  returning it to learning mode to be relearned.

### Decks

A _deck_ is a group of cards. You can place cards in different decks to
study parts of your card collection instead of studying everything at
once. Each deck can have different settings, such as how many new cards
to show each day, or how long to wait until cards are shown again.

Decks can contain other decks, which allows you to organize decks into a
tree. Anki uses double colons ("::") to show different levels within the deck tree. For example, a deck called
"Chinese::Hanzi" refers to a "Hanzi" deck, which is part of a "Chinese"
deck. If you select "Hanzi", then only the Hanzi cards will be shown; if
you select "Chinese", then all the Chinese cards will be shown, including the Hanzi cards.

> To place decks under a tree, you can either name them with double colons between
> each level, or drag and drop them within the deck list. Decks that have
> been placed under a tree are often called "subdecks", and top-level decks are called "parent decks".

Author: How about "within" instead of "into"? I'll apply this if no qualms.

```suggestion
To place decks within a tree, you can either name them with double colons between
each level, or drag and drop them within the deck list. Decks that have
been placed inside another deck are often called "subdecks", and top-level decks are called "parent decks".
```

    A: O

After confirming that you are correct, you tell Anki how well you
remembered the answer, and Anki will choose when to show you the card again. For example, Anki might decide to show you the card again in 3 days. In this case, we say the card now has a 3 day interval.

> #### States of Cards

Reviewer: ```suggestion
#### Card States
```

    - **Young:** A young card is one that has an interval of less than 21 days.
    - **Mature:** A mature card is one that has an interval of 21 days or greater.

> - **Relearn:** Cards that have failed in review state, and have returned to learning state to be relearned.

Reviewer: ```suggestion
- **Relearn:** Cards that have failed in the review state, and have returned to the relearning state to be learned again.
```

Anki solves these problems by allowing you to split the content of your
cards up into separate pieces of information. You can then tell Anki

> which pieces of information you want on each card, and Anki takes care of creating the cards for you and updating them if you make any

Reviewer 2: ```suggestion
which pieces of information you want on each card, and Anki takes care of creating the cards for you, and updating them if you make any
```

Anki comes with some standard
note types included. These note types are provided to make Anki easier for
new users, but in the long run it's recommended you create your own note
types specifically for the content you are learning. The standard note types are:

- **Basic**\
  Has "Front" and "Back" fields, and will create one card. Text you enter in
  "Front" will appear on the front of the card, and text you enter in "Back"
  will appear on the back of the card.

- **Basic (and reversed card)**\
  Like "Basic", but creates two cards for the text you enter: 
  front→back and back→front.

- **Basic (optional reversed card)**\
  Like "Basic", but has a third field called "Add Reverse". If you enter any text into
  that field, a reversed card (back→front) will also be created. For details, see the [Cards and Templates](templates/intro.md) section.

- **Basic (type in the answer)**\
  This is essentially "Basic", with an extra text box on the front where you
  can type your answer in. After flipping to the back, you can check and
  compare your input with the actual answer. For details, see the
  [Checking Your Answer](templates/fields.md#checking-your-answer) section.

- **Cloze**\

>   A note type which makes it easier to select text and turn it into a cloze

Reviewer: ```suggestion
  A note type that makes it easy to select text and turn it into a cloze
```
"Easier" sounds wrong, because we're not comparing to anything -- it's either possible (with this note type), or impossible (without it). If you don't want to say "easy," perhaps -- 

```suggestion
  A note type that allows you to select text and turn it into a cloze
```
And switching from "which" to "that" because it's necessary clause.

Reviewer: This one is hidden, but not resolved.

Anki solves these problems by allowing you to split the content of your
cards up into separate pieces of information. You can then tell Anki
which pieces of information you want on each card, and Anki will take

> care of creating the cards for you.

Author: This is rather obvious I think. I don't think users should be distracted from the topic at hand. If it needs documentation, it should be in Adding/Editing section.

> care of creating the cards for you.

Reviewer: It's just such an integral part of the workflow -- not being stuck with your first decision -- and I don't think it goes without saying. I still think the original language is best, but perhaps there's a compromise?

```suggestion
care of creating and updating the cards for you.
```

> care of creating the cards for you.

Author: I want you to read the whole thing, it's not about updating at all. This is just confusing that we're trying hard to add this info here. We also repeat the updating part later in the same section.

> care of creating the cards for you.

Reviewer: I have read it, I assure you. I know what the other reference is, and I don't think it eliminates the use for mentioning it here. I'm here from the past to tell you that having persistently updated cards is really quite a novel concept.

Imagine we want to study French vocabulary, and we want to include a

> page number on the back of each card. We want the back of our cards to look like

Author: In the beginning example chem example, users are shown the back of a card which includes both question and answer. It's the answer side here.

> page number on the back of each card. We want the back of our cards to look like

Reviewer: I see the parallel you're drawing, but here, we're talking about rearranging the fields of information to create both the question and answer sides of the card. After all, this wouldn't be that great of a feature if you could only use it to change things on the _back_.

> page number on the back of each card. We want the back of our cards to look like

Author: Then it shouldn't be in the same code block because this actually tripped me up. Even then, talking about both templates at the same time is confusing for the reader.
 I also don't think the goal here is to let people know about all the things you can do in Anki, but to give them guidance instead to be able to discover and use it for themselves. Both the templates are mentioned and people understanding one of them is enough.

> page number on the back of each card. We want the back of our cards to look like

Reviewer: I think an average user is much more concerned with what happens on the _front_ of their cards, than the back. But this clearly denotes a Q side and an A side.

they’d look like this:

    French: Bonjour
    English: Hello
    Page: 12

In Anki, this collection of related information is called a _note_ and each piece of information is contained in a _field_. In this example, the note
has three fields: "French", "English", and "Page".

To add and edit fields, click the "Fields…​" button while adding or
editing notes. For more information on fields, please see the
[Customizing Fields](editing.md#customizing-fields) section.

### Card Types

In order for Anki to create cards based on our notes, we need to give it
a blueprint that says which fields should be displayed on the front or
back of each card. This blueprint is called a _card type_. Each type of
note can have one or more card types; when you add a note, Anki will
create one card for each card type.

All card types have two _templates_, one for the question and one for the
answer. In the previous French example, we wanted the back of our recognition card to
look like this:

    Q: Bonjour
    A: Hello
       Page #12

To do this, we can set the answer template to:

    Q: {{French}}
    A: {{English}}<br>
       Page #{{Page}}

> In card templates, field names are wrapped in double curly brackets like `{{French}}` or `{{English}}`. Anki replaces those with the actual text the field contain. This is known as ["Field replacement"](/field-replacement.md). Text
> not surrounded by curly brackets appears the same on each card. For
> example, we don't need to enter "Page \#" in every card; it’s added automatically to every card. The `&lt;br&gt;` tag is

Author: I suggest we remove the word field and the semicolon to simply say "we don't need to enter "Page \#" on every note as it's added automatically to every card."

 It looks a bit weird to have "every note" and "every card" together but I believe this is better.

```suggestion
In card templates, field names are wrapped in double curly brackets, like `{{French}}` or `{{English}}`. Anki replaces those with the actual text the fields contain. This is called a ["Field replacement"](templates/fields.md). Text not wrapped in double curly brackets appears the same on each card. For example, we don't need to enter "Page \#" on every note as it’s added automatically to every card. The `&lt;br&gt;` tag is
```

> In card templates, field names are wrapped in double curly brackets like `{{French}}` or `{{English}}`. Anki replaces those with the actual text the field contain. This is known as ["Field replacement"](/field-replacement.md). Text
> not surrounded by curly brackets appears the same on each card. For
> example, we don't need to enter "Page \#" in every card; it’s added automatically to every card. The `&lt;br&gt;` tag is

Reviewer: How about this? 
```suggestion
In card templates, field names are wrapped in double curly brackets, like `{{French}}` or `{{English}}`. Anki replaces those with the actual text the fields contain. This is called a ["Field replacement"](templates/fields.md). Text not wrapped in double curly brackets appears the same on each card. For example, we won't need to add "Page \#" on every note because the template will add it automatically to every card. The `&lt;br&gt;` tag is
```

The production card's templates will also work in a similar way:

    Q: {{English}}
    A: {{French}}<br>
       Page #{{Page}}

After a card type has been created, every time you add a new note, a card
will be created based on that card type. Card types make it easy to keep
the formatting of your cards consistent and can greatly reduce the
amount of effort involved in adding information. They also mean Anki can
ensure related cards don't appear too close to each other, and they
allow you to fix a typing mistake or factual error once and have all the
related cards updated at once.

To add and edit card types, click the "Cards…​" button while adding or
editing notes. For more information on card types, please see the [Cards and Templates](templates/intro.md) section.

### Note Types

Anki allows you to create different types of notes for different
material. Each type of note has its own set of fields and card types.
It's a good idea to create a separate note type for each broad topic
you’re studying. In the previous French example, we might create a note
type called "French" for that. If we wanted to learn capital cities, we

> could create a separate note type for that as well, with fields such as

Author: I think it's unnecessary, and to my ears "world capital cities" sounds weird (probably because here we say country capital/state capital).

Let's also keep the "as well"; it flows well if you read the whole thing.

> could create a separate note type for that as well, with fields such as

Reviewer: "capital cities of the world"?

I think we don't need both "separate" and "as well" -- both of them are doing the same job.

> could create a separate note type for that as well, with fields such as

Author: I don't see the point in perfectionism.

I don't know, "seperate" and "as well" are different things for me.

> type called "French" for that. If we wanted to learn capital cities, we
> could create a separate note type for that as well, with fields such as

Author: ```suggestion
type called "French" for that. If we wanted to learn capital cities, we
could create a note type for that as well, with fields such as
```

"Country" and "Capital City".

Anki comes with some standard
note types in it. These note types are provided to make Anki easier to use for
new users, but in the long run it's recommended you create your own note
types for the content you are learning. The standard note types are as
follows:

- **Basic**\
  Has "Front" and "Back" fields, and will create one card. Text you enter in

>   "Front" will appear on the front of the card, and text you enter in "Back"

Author: It also looks there there is no space between `in` and `"Back"` but the page looks fine in my repo. Probably not an issue.

>   "Front" will appear on the front of the card, and text you enter in "Back"

Reviewer: Okay, good! It will be easy to spot when we have a clean copy anyway!

  will appear on the back of the card.

- **Basic (and reversed card)**\
  Like "Basic", but creates two cards for the text you enter: one from
  front→back and one from back→front.

- **Basic (optional reversed card)**\
  Like "Basic", but has a third field called "Add Reverse". If you enter any text into

>   that field, a reverse card will be created. For details, see the [Cards and Templates](templates/intro.md) section.

Author: How about a reverse card?

- **Basic (type in the answer)**\
  This is essentially "Basic", with an extra text box on the front where you
  can type your answer in. After flipping to the back, you can check and

>   compare your input with the answer. For details, see the

Author: I would want people to experience it for themselves. For some of the readers here, this sentence might make them think Anki answers the card for them. People still make the manual decision of what to pick, so this is a better choice.

>   compare your input with the answer. For details, see the

Reviewer: But this also makes it sound like the sort of "just let me compare it myself" simple carry-over field that folks ask for and we don't have as a built-in feature. What if we blame this one on the passive voice? 

```suggestion
  can type your answer in. When you reveal the back, Anki will show you any differences between 
  your input and the actual answer. For details, see the
```
[Apparently this text is outdated now, so I copied this from the current commit.]

  [Checking Your Answer](templates/fields.md#checking-your-answer) section.

- **Cloze**\
  A note type which makes it easy to select text and turn it into a cloze
  deletion (e.g. "Humans landed on the moon in \[…​\]" → "Humans landed on the
  moon in 1969"). For details, see the [cloze deletion](editing.md#cloze-deletion) section.

- **Image Occlusion**\
  Like the cloze note type, but it works with images instead of text,
  which is especially useful when studying material that heavily relies on images,
  such as anatomy and geography. For details, please see the [Image Occlusion](editing.md#image-occlusion)
  section of the manual. 

To add your own note types and modify existing ones, you can use Tools →
Manage Note Types from the main Anki window.

Notes and note types are common to your whole collection rather than
limited to an individual deck. This means you can use different
notes types in a single deck, or have cards generated from the
 same note in different decks. When you add notes using the
Add window, you can select what note type to use and what deck to use,
and these choices are completely independent of each other. You can also

> change the note type of some notes [after you've already created them](browsing.md).

Reviewer: ```suggestion
[change the note type of notes](browsing.md#notes) after you've already created them.
```

### Collection

Your _collection_ is all the material stored in Anki: your cards,
notes, decks, note types, deck options, and so on.

## Shared Decks

You can watch a video about [Shared Decks and Review
Basics](http://www.youtube.com/watch?v=QS2G-k2hQyg&yt:cc=on) on YouTube.

The easiest way to get started with Anki is to download a deck of cards
someone else has shared:

1. Click the "Get Shared" button at the bottom of the deck list.

2. When you've found a deck you're interested in, click the "Download"
   button to download a deck package.

3. Double-click the downloaded package to import it into Anki, or go to
   File → Import.

Note: It’s not currently possible to add shared decks

> directly to your AnkiWeb account. Please first import them to the
> desktop app, then synchronize to upload the decks to AnkiWeb.

Reviewer: ```suggestion
directly to your AnkiWeb account. You need to first import them to the
desktop app, AnkiMobile, or AnkiDroid, then synchronize to upload the decks to AnkiWeb.
```

Creating your own deck is the most effective way to learn a complex
subject. Subjects like languages and the sciences can’t be understood
simply by memorizing facts — you need explanation and context to
learn them effectively. Furthermore, inputting the information yourself
forces you to decide what the key points are, leading to a better
understanding.

If you are a language learner, you may be tempted to download a long
list of words and their translations, but this won’t teach you a
language any more than memorizing scientific equations will teach you

> astrophysics. To learn properly, we believe you need textbooks, teachers, or

Reviewer: ```suggestion
astrophysics. To learn properly, you also need textbooks, teachers, or
```
Using "we believe" undercuts the statement. If it's not true and accurate, it's not worth saying.

Author: It's important to emphasise that opinions are subjective especially because people are divided on "how best to learn". For example, I personally don't think you need textbook/teachers, real world sentences are important but my beliefs are more nuanced.

Reviewer: ```suggestion
astrophysics. To learn properly, you might need textbooks, teachers, or
```

Anki allows you to create different types of notes for different
material. Each type of note has its own set of fields and card types.
It’s a good idea to create a separate note type for each broad topic
you’re studying. In the above French example, we might create a note
you’re studying. In the previous French example, we might create a note
type called “French” for that. If we wanted to learn capital cities, we
could create a separate note type for that as well, with fields such as
“Country” and “Capital City”.

> When Anki checks for duplicates, it only compares other notes of the

Author: This is documented elsewhere, so I removed it.
